tests/test_cli.py

```diff
@@ -958,4 +978,7 @@ def test_metadata_validate_no_file(mockargs):
 
 
 def test_metadata_template():
-    metadata_template()
+    loc = tempfile.TemporaryDirectory()
```

Comment:
As above about tempfile. I've added a temporary filesystem fixture in the E2E stuff, so I think we can come back and address this later TBH - I think we'll want to merge both in before addressing tempfile stuff to avoid effort duplication.

Reply:
OK let's address the tempfile/tempdir things in a new issue/PR, but do that as a priority - we don't want to be flooding test systems accidentally.